packages/manager/modules/pci/src/projects/new/config/index.html

```diff
@@ -39,6 +39,7 @@ <h1 data-translate="pci_project_new_config_title"></h1>
 
     <!-- contracts -->
     <ovh-contracts-summary
+        data-inline-items="true"
```

Comment:
Could we handle this case 
```suggestion
        data-inline-items
```
like it is commonly done for boolean attributes

Reply:
Thanks Marie for this suggestion.
Yep it could be interesting to have this behavior.

For the moment we can go with this implementation (fully functional) and once i will have more time i will dig more how we can achieve it.